src/drawers/errorBarDrawer.ts

```diff
@@ -0,0 +1,77 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module _Drawer {
+  export class ErrorBar extends Element {
+    public static ERROR_BAR_CLASS = "error-bar";
+    public static ERROR_BAR_MIDDLE_CLASS = "error-bar-middle";
+    public static ERROR_BAR_LOWER_CLASS = "error-bar-lower";
+    public static ERROR_BAR_UPPER_CLASS = "error-bar-upper";
+
+    private _errorTickLength = 20;
+    private _isVertical: boolean;
+
+    constructor(key: string, isVertical: boolean) {
+      super(key);
+      this.svgElement("g");
+      this._isVertical = isVertical;
+    }
+
+    protected _enterData(data: any[]) {
+      var errorBars = this._getRenderArea().selectAll(this._svgElement).data(data);
+      errorBars.enter().append("g").classed(ErrorBar.ERROR_BAR_CLASS, true)
+          .each(function(d) {
+            d3.select(this).append("line").classed(ErrorBar.ERROR_BAR_LOWER_CLASS, true);
+            d3.select(this).append("line").classed(ErrorBar.ERROR_BAR_MIDDLE_CLASS, true);
+            d3.select(this).append("line").classed(ErrorBar.ERROR_BAR_UPPER_CLASS, true);
+          });
+      errorBars.exit().remove();
+    }
+
+    protected _drawStep(step: AppliedDrawStep) {
+      super._drawStep(step);
+
+      var attrToProjector = <AttributeToAppliedProjector>_Util.Methods.copyMap(step.attrToProjector);
+
+      var xProjector = attrToProjector["x"];
+      var yProjector = attrToProjector["y"];
+      var lowerProjector = attrToProjector["lower"];
+      var upperProjector = attrToProjector["upper"];
+
+      var halfTickLength = this._errorTickLength / 2;
+
+      var minProjector = this._isVertical ? (d: any, i: number) => xProjector(d, i) - halfTickLength :
+                                            (d: any, i: number) => yProjector(d, i) - halfTickLength;
+      var maxProjector = this._isVertical ? (d: any, i: number) => xProjector(d, i) + halfTickLength :
+                                            (d: any, i: number) => yProjector(d, i) + halfTickLength;
+
+      if (this._isVertical) {
+        this.setProjectorsForLine(ErrorBar.ERROR_BAR_LOWER_CLASS, minProjector, lowerProjector, maxProjector, lowerProjector);
+        this.setProjectorsForLine(ErrorBar.ERROR_BAR_MIDDLE_CLASS, xProjector, lowerProjector, xProjector, upperProjector);
+        this.setProjectorsForLine(ErrorBar.ERROR_BAR_UPPER_CLASS, minProjector, upperProjector, maxProjector, upperProjector);
+      } else {
+        this.setProjectorsForLine(ErrorBar.ERROR_BAR_LOWER_CLASS, lowerProjector, minProjector, lowerProjector, maxProjector);
+        this.setProjectorsForLine(ErrorBar.ERROR_BAR_MIDDLE_CLASS, lowerProjector, yProjector, upperProjector, yProjector);
+        this.setProjectorsForLine(ErrorBar.ERROR_BAR_UPPER_CLASS, upperProjector, minProjector, upperProjector, maxProjector);
+      }
```

Comment:
I believe you need five distinct projectors for that, which I think is considerably more confusing than the way it is right now, because you have to think about what to call each part of the lower/upper parts of the errorBar. 

![image](https://cloud.githubusercontent.com/assets/1175041/6755965/f4ce58e6-cee3-11e4-8b22-254a02109b6f.png)